server/interactions/interactions.js

```diff
@@ -0,0 +1,27 @@
+const routes = require("express").Router();
+const axios = require("axios");
+const token = require("../../config");
+const url = "https://app-hrsei-api.herokuapp.com/api/fec2/hr-atx";
```

Comment:
for constants in the truest sense of the word (static, immutable constants), a popular convention is to use all caps

```node.js
const URL = "Some Static Unchanging Value"
```

![image](https://user-images.githubusercontent.com/12129271/143925021-18b9f188-a45b-49c2-80aa-ce0589b78fe1.png)